.github/workflows/main.yaml

```diff
@@ -53,3 +60,25 @@ jobs:
           fail_ci_if_error: true
           files: ./coverage.xml
           flags: unittests
+  build-and-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+    needs:
+      - unit-tests
+      - integration-tests
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install pypa/build
+        run: python -m pip install build --user --upgrade
```

Comment:
I'd add a second requirement file requirements-build.txt or something

Reply:
Do you mean we should replace `python -m pip install build --user --upgrade` by something like this:
`pip install -r requirements-build.txt`?

Reply:
exactly. We can then have the same for linting etc, so we and users only install the things that are really needed

Reply:
if we don't need it, let's just get rid of the line :)